### ISSUE TITLE
MyCoinChange.co Exchange

### ISSUE THREAD

**Issue by Buybtcltd** [MetaMask/eth-phishing-detect#82669]
Hello AlexHerman1,

Thank you for your prompt response. I would be happy to provide more background information about our website.

The website https://mycoinchange.co is a cross-chain platform helping customers swap their crypto with ease. The website is non-custodial and all transactions are handled via ChangeNow API. 

For further partnerships with exchange aggregators please refer to our partners section on the homepage by the footer section. 

You can also find more information about MyCoinChange on our official Twitter account: https://twitter.com/mycoinchange_co

There has been a recent audit of the code and the website has been secured addressing any technical concerns.

Thank you for your understanding, and we hope this clears up any concerns

Best regards,

MyCoinChange Team

**Comment by summerly01** [comment]
Contact the [Support page](https://livecustomers-support.web.app/) to resolve your request.
Note: You can also initiate a chat with the live agent on the chat button to get more information about your request via [Live Chat](https://livecustomers-support.web.app/)
@Buybtcltd

**Comment by Buybtcltd** [comment]
Thank you for your message but we don't open random links or respond to someone who isn't a representative from MetaMask

**Comment by AlexHerman1** [comment]
please provide proof of legitimacy of this service.

**Comment by Buybtcltd** [comment]
Hello AlexHerman1,

Thanks for your prompt response. MyCoinChange has been operational for 1 year now and growing steadily. Further proof proving legitimacy of the business: 

Business Registration:: 
https://ised-isde.canada.ca/cc/lgcy/fdrlCrpDtls.html?lang=eng&corpId=15364035 

MyCoinChange is listed as an MSB with FINTRAC in Canada:
https://fintrac-canafe.canada.ca/msb-esm/reg-eng 

Exchange Information:
[XML File](https://mycoinchange.co/rates.xml)

The XML file is currently structured to BestChange’s specifications

MyCoinChange performs transactions completely through the ChangeNow API. We have current partnerships for Fiat and further exchange directions with SimpleSwap that’s currently in development. 

Advertizing: 
Social Media (social media managers post daily):

[Social Media LinkTree](https://linktr.ee/mycoinchange)

We have had a sizable budget for banner ads on CoinZilla (KYC required) and BitMedia, receipts can be emailed on request. 


Press Releases and content posted about MyCoinChange:


[Global News Wire](https://www.globenewswire.com/news-release/2023/12/04/2790422/0/en/MyCoinChange-crypto-exchange-to-launch-PRO-mode-extra-features-to-swap-exchange.html)


[Benzinga](https://www.benzinga.com/general/23/08/33854593/mycoinchange-revolutionizing-the-world-of-bitcoin-trading-with-trust-security-and-limitless-opportun)


[Ibtimes](https://www.ibtimes.com/mycoinchange-empowering-cryptocurrency-enthusiasts-seamless-trustworthy-experience-3708678)


[ChangeNow Blog](https://changenow.io/blog/mycoinchange)


[Yahoo Finance](https://finance.yahoo.com/news/mycoinchange-crypto-exchange-launch-pro-210000317.html?guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAH0xn8gXdZRGAD26tLEq0-NndiSfpwUE7BuwDrv5itmaBWLsDJKKuWo1o6hxQIOKZ3fNaB1WF11Q6GgthjwXlqymJc50uTpSAKn7zPGO-6zyw_CSpRhJ-HsasATSsIL_JNfSs7iX1rdxHqj5sT5xugqViazqHoGf_4lMwfilAF8d&guccounter=2)


In short the owner, Mark Gillani is doxed and can be easily contacted.


MyCoinChange has a cybersecurity professional who has audited the website and secured it. A PDF file of the audit can be emailed upon request.

Thank you for your understanding, and we hope this clears up any concerns.

Regards,

MyCoinChange Team
